README.rst

```diff
@@ -72,6 +72,10 @@ Documentation
 Usage and API documentation can be found at
 `<https://pythonhosted.org/nengo/>`_.
 
+To build the documentation yourself, `follow the instructions in the Developer
```

Comment:
Here, I would say that the documentation can be built with `python setup.py build_sphinx`, and requires the packages in requirements-test.txt and requirements-docs.txt, as well as Pandoc. That gives readers a quick one-sentence summary, and then the link can provide more detailed information.

Really, the most important part is having the build command written here so that I can refer to it easily. Even if we just have that along with the link, I'll be happy.